2-projects/1-todo-list/script.js

```diff
@@ -1,25 +1,93 @@
-function populateTodoList(todos) {
-  let list = document.getElementById("todo-list");
-  // Write your code to create todo list elements with completed and delete buttons here, all todos should display inside the "todo-list" element.
-}
-
-// These are the same todos that currently display in the HTML
+function toggleIsComplete(todo) {
+    todo.completed = !todo.completed;
+  }
+  
+  function redrawTodo(li, isComplete) {
+    if (isComplete) {
+      li.style.textDecoration = "line-through";
+    } else {
+      li.style.textDecoration = null;
+    }
+  }
+  
+  function deleteTodo(li) {
+    document.querySelector("ul").removeChild(li);
```

Comment:
this will remove the first `ul` that exists in the page, how could you make this more specific to the todo list element you want?